hutch_python/happi.py

```diff
@@ -30,7 +30,8 @@ def get_happi_objs(
     light_ctrl: LightController,
     endstation: str,
     load_level: DeviceLoadLevel = DeviceLoadLevel.STANDARD,
-    exclude_devices: list[str] = None
+    exclude_devices: list[str] = None,
+    additional_devices: list = None
```

Comment:
```suggestion
    exclude_devices: Optional[list[str]] = None,
    additional_devices: Optional[list] = None
```
This wasn't properly hinted before but while we're here we can be more accurate